README.md

```diff
@@ -82,31 +89,31 @@ created by github.com/google/gops/agent.Listen
 # ...
 ```
 
-#### $ gops memstats \<pid\>
+#### $ gops memstats \<target\>
```

Comment:
We should use the one or the other notation:

gops memstats (\<pid\>|\<addr\>)

Use <addr> for the agent URL.